### ISSUE TITLE
confused on dockpanel styling

### ISSUE THREAD

**Issue by vrmerlin** [phosphorjs/phosphor#247]
I'm using the dockpanel widget for a project, and am confused on what styling to use. I see this:

https://github.com/phosphorjs/phosphor/tree/master/packages/widgets/style

and I see this:

https://github.com/phosphorjs/phosphor/tree/master/examples/example-dockpanel/style

If I just reference the styles from the first URL, then the tabs don't show up and things don't behave right. If I copy/paste the dockpanel.css and tabbar.css content from the example-dockpanel into my own style sheet (in addition to the widget package styles), then it works.  Why aren't the widget styles sufficient to get a default appearance? The dockpanel seems useless without pulling in the example styles, which seems strange. Maybe I'm missing something...

Thanks,
John

**Comment by sccolbert** [comment]
The styles which are included in the core packages are *required*. That is,
if you don't include these, Phosphor will fail to function properly.
However, these core styles are not *sufficient* for a fully styled app.

Phosphor aims to be a fully generic framework which can be used to build a
variety of applications. To that end, it would be impossible for Phosphor
to define a default style which would please everyone, which would leave
users having to *undo* the default style rules, which is very tedious. We
find it better to just not provide a default style at all.

In the near future, I plan to add separate theme packages, which you can
use to style your app (the examples will use these as well). We've just had
higher priority things to deal with up til now.

TL;DR the packages styles are *necessary* but not *sufficient* for a fully
styled app. In the future we'll add theme packs which provide opinionated
styles.

**Comment by vrmerlin** [comment]
Ok, got it -- thanks for clarifying! We look forward to the themes.  :-)
